test/pytest/test_model_control_mode.py

```diff
@@ -0,0 +1,124 @@
+import os
+import tempfile
+from pathlib import Path
+
+import pytest
+import requests
+import test_utils
+
+ROOT_DIR = os.path.join(tempfile.gettempdir(), "workspace")
+REPO_ROOT = os.path.dirname(os.path.abspath(__file__))
+
+expected_output = {
+    "code": 405,
+    "type": "MethodNotAllowedException",
+    "message": "Requested method is not allowed, please refer to API document.",
+}
+
+
+@pytest.fixture(scope="module")
+def setup_torchserve():
+    MODEL_STORE = os.path.join(ROOT_DIR, "model_store/")
+
+    Path(test_utils.MODEL_STORE).mkdir(parents=True, exist_ok=True)
+
+    test_utils.start_torchserve(
+        no_config_snapshots=True, models="mnist=mnist.mar", mode="none"
```

Comment:
The interface should be 
```
def start_torchserve(..., model_control_enabled=True):
```
so when you setup a test that want the new behavior you use:
```
test_utils.start_torchserve(..., model_control_enabled=False)
```